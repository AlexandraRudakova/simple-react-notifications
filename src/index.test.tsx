import React from "react";
import Hello from "./index";
import { shallow } from "enzyme";

describe("<Hello />", () => {
  it("should render correctly with a text prop", () => {
    const component = shallow(<Hello text="Alex" />);

    expect(component).toMatchSnapshot();
  });
});
