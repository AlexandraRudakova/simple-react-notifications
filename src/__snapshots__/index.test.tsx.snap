// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hello /> should render correctly with a text prop 1`] = `
<div
  style={
    Object {
      "color": "red",
    }
  }
>
  Hello 
  Alex
</div>
`;
